src/content/docs/tutorial-customer-experience/bottom-funnel-analysis.mdx

```diff
@@ -0,0 +1,190 @@
+    <Step>
+### Create a scripted monitor for the bottom of the funnel
+
+Make sure you have a [scripted monitor](/docs/synthetics/synthetic-monitoring/scripting-monitors/introduction-scripted-browser-monitors/) for each path through the bottom of the funnel. The goal is to make sure your bottom-of-the-funnel services are working around the clock.
+
+For example, you may have a checkout flow that calls a different payment API depending on the customer's payment preferences.        
+    </Step>
+    <Step>
+### Import the bottom-of-the-funnel dashboard
+
+Follow the instructions in the [bottom-of-the-funnel analysis quickstart](https://newrelic.com/instant-observability/customer-experience-bottom-funnel-analysis/8d339aa7-69e3-4efa-abc2-9eba600db91c) to install the dashboard and configure it.
+    </Step>
+    <Step>
+### Capture current performance
+
+<img
+  title="Bottom-of-the-funnel KPI tracking"
+  alt="Bottom-of-the-funnel KPI tracking"
+  src={cxBotfaKpiTracking}
+/>
+
+1. Follow the instructions in our [bottom-of-the-funnel analysis README on GitHub](https://github.com/newrelic/oma-resource-center/blob/main/src/content/docs/oma/value-drivers/customer-experience/use-cases/bottom-of-the-funnel-analysis/README.md).
+2. Use the dashboard from the previous step to understand the bottom-of-the-funnel performance.
+3. Create a plan to improve KPIs that don't meet target values, as well as reduce revenue at risk.
+    </Step>
+    <Step>
+### Improvement process
+Whether you have a dedicated initiative to improve performance or classifying as ongoing maintenance, you need to track your progress at the end of every sprint. For detailed information, see:
+
+* [Improve uptime](/docs/new-relic-solutions/observability-maturity/customer-experience/cx-improve-web-uptime)
+* [Improve page load performance](/docs/new-relic-solutions/observability-maturity/customer-experience/cx-improve-page-load)
+* [Improve AJAX performance](/docs/browser/browser-monitoring/browser-pro-features/ajax-page-identify-time-consuming-calls)
+    </Step>
+</Steps>
+
+
+<Callout variant="tip">
+A common question is: Should I apply segmentation (breaking out performance into cohorts, such as region and device type)? It depends on your needs, but segmentation is a good idea if:
+* Your organization wants to address a target audience from a particular cohort that you can segment by using either custom attributes or data that is already available in New Relic.
+* You're already familiar with bottom-of-the-funnel analysis and there's a significant enough difference in performance among different cohorts to warrant tracking and/or developer focus.
+</Callout>
+
+### Best practices going forward
+
+After you've followed the procedure above, it's a good idea to re-analyze the bottom-of-the-funnel whenever you can. Each organization will have different needs, and so the procedures you use will be unique to your organzation. However, you can keep the following suggestions in mind:
```

Comment:
I think you should rewrite this conclusion to be stronger. Recap what someone has accomplished, how it will help them going forward, and highlight these best practices and why they're important.

Reply:
Good call, I'll make sure it's done. Thank you!